### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by jackdavidweber**
Mostly filled in the template which involved changing text, playing with features and uploading images. Also used spotify player rather than just listing my favorite songs.

https://screenshot.googleplex.com/iONZeVXzs97

https://screenshot.googleplex.com/UQjRQomCTh1.png

### CODE REVIEW

File: portfolio/src/main/webapp/template.html [hunk 1]
```diff
@@ -68,73 +68,39 @@ <h2 class="heading">Latest Projects</h2>
                             <div class="item featured text-center">
                                 
                                 <div class="featured-image has-ribbon">
-                                    <a href="https://themes.3rdwavemedia.com/bootstrap-templates/startup/launch-bootstrap-4-template-for-saas-businesses/" target="_blank">
-                                    <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/project-featured.jpg" alt="project name" />
+                                    <a href="#" target="_blank">
+                                    <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/GradTracks.png" alt="GradTracks" />
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html:73]: Per the style guide, break long lines to improve readability here and everywhere:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/template.html [hunk 2]
```diff
@@ -68,73 +68,39 @@ <h2 class="heading">Latest Projects</h2>
...
                                     </a>
                                     <div class="ribbon">
                                         <div class="text">New</div>
                                     </div>
                                 </div>
                                 
-                                <h3 class="title mb-3"><a href="https://themes.3rdwavemedia.com/bootstrap-templates/startup/launch-bootstrap-4-template-for-saas-businesses/" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a></h3>
+                                <h3 class="title mb-3"><a href="#" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a></h3>
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html:80]: Per the style guide, new lines for every block and indent every child element here and everywhere:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/template.html [hunk 3]
```diff
@@ -154,44 +155,41 @@ <h2 class="heading sr-only">Basic Information</h2>
                     <div class="section-inner shadow-sm rounded">
                         <h2 class="heading">Skills</h2>
                         <div class="content">
-                            <p class="intro">
-                                Intro about your skills goes here. Keep the list lean and only show your primary skillset. You can always provide a link to your Linkedin or Github profile so people can get more info there.</p>
-                            
                             <div class="skillset">
-                               
+
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html:169]: Nit:
In some spaces you have new lines between the first nested tag and in most places you don't. Pick a strategy and try to be consistent everywhere:
https://google.github.io/styleguide/htmlcssguide.html#Parting_Words

**Review by Danish21 — APPROVED** [review]
Nice work. Looks great. 

LGTM aside from the style guide comments.

**Review by augerm — COMMENTED** [review]
Could we get a screenshot of how it looks on mobile? In chrome dev tools you can emulate mobile and take a screenshot from there.

**Pull request merged**